packages/components/entity-list/src/EntityListItem/EntityListItem.tsx

```diff
@@ -90,7 +92,14 @@ export interface EntityListItemProps extends CommonProps {
    *
    * Note: 'entry' and 'asset' are @deprecated but supported in v1.x for backwards compatibility
    */
-  entityType?: 'Entry' | 'Asset' | 'entry' | 'asset' | 'Release' | 'Page';
+  entityType?:
+    | 'Entry'
+    | 'Asset'
+    | 'entry'
+    | 'asset'
+    | 'Release'
+    | 'Page'
+    | 'Experience';
```

Comment:
is there also the possibility of it beeing lower case 'experience' like we have it for asset and entry?